frontend/admin-ui/package.json

```diff
@@ -60,8 +60,8 @@
     "start:integration:dev": "concurrently \"yarn cypress:integration:dev\" --kill-others \"yarn start:dev\" --success first",
     "storybook": "start-storybook -p 7006",
     "test": "echo \"See package.json => scripts for available tests.\" && exit 0",
-    "test:unit": "jest --updateSnapshot",
-    "test:unit:ci": "jest --updateSnapshot --ci --reporters=default --reporters=jest-junit",
```

Comment:
If that option is passed in on the ci, then we're screwed up, as it updates the snapshot files with potentially unwanted modifications.